### PR TITLE
chore: bump open-clip-torch to v2.8.0

### PR DESCRIPTION
**Pull request by ZiniuYu**
This PR adapts recent changes introduced in open-clip-torch v2.8+

### CODE REVIEW

File: server/clip_server/model/model.py [hunk 1]
```diff
@@ -111,6 +120,8 @@ class CLIPVisionCfg:
     patch_size: int = 16
     image_size: Union[Tuple[int, int], int] = 224
     ls_init_value: Optional[float] = None  # layer scale initial value
+    patch_dropout: float = 0.0  # what fraction of patches to dropout during training (0 would mean disabled and no patches dropped) - 0.5 to 0.75 recommended in the paper for optimal results
```
Comment (jemmyshin) [inline server/clip_server/model/model.py:123]: why we need dropout during inference? It's not training process

**Review by jemmyshin — CHANGES_REQUESTED** [review]

File: server/clip_server/model/model.py (hunk 1 above)
Comment (ZiniuYu) [reply server/clip_server/model/model.py:123]: disabled by default

Comment (jemmyshin) [reply server/clip_server/model/model.py:123]: Then we can remove this args if we don't need it

File: tests/test_server.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def test_server_download(tmpdir):
     download_model(
         url='https://docarray.jina.ai/_static/favicon.png',
         target_folder=tmpdir,
-        md5sum='a084999188f4290e2654aec43207ff2e',
+        md5sum='66ea4817d73514888dcf6c7d2b00016d',
```
Comment (jemmyshin) [inline tests/test_server.py:15]: is there any method we can auto-calculate the `md5` in order not to block by this image chaning?

Comment (ZiniuYu) [reply tests/test_server.py:15]: no

**Review by jemmyshin — APPROVED** [review]
lgtm

File: tests/test_server.py (hunk 2 above)
Comment (jemmyshin) [reply tests/test_server.py:15]: https://github.com/jina-ai/clip-as-service/blob/0b293ec834e80f7335aa625d683904594373a607/server/clip_server/model/pretrained_models.py#L124
